packages/presentation/lib/screen/home/home_screen.dart

```diff
@@ -68,79 +78,77 @@ class _HomeScreenState extends BlocScreenState<HomeScreen, HomeBloc>
                   ),
                   actions: [
                     IconButton(
-                      onPressed: () {},
-                      icon: const Icon(
-                        Icons.search,
-                        size: 40,
-                      ),
+                      onPressed: () => {},
+                      icon: SvgPicture.asset(ImagesPath.searchIcon),
                       color: Colors.white,
                     ),
                   ]),
-              body: SafeArea(
-                child: Column(
-                  children: [
-                    TabBarContainer(tabController: tabController),
-                    Flexible(
-                      child: GridView.builder(
-                        gridDelegate:
-                            const SliverGridDelegateWithMaxCrossAxisExtent(
-                                maxCrossAxisExtent: 200,
-                                childAspectRatio: 3 / 2,
-                                crossAxisSpacing: 20,
-                                mainAxisSpacing: 20),
-                        itemCount: 10,
-                        itemBuilder: (BuildContext ctx, index) {
-                          return Container(
-                            alignment: Alignment.center,
-                            decoration: BoxDecoration(
-                              color: Colors.amber,
-                              borderRadius: BorderRadius.circular(15),
-                            ),
-                            child: const Text("name"),
-                          );
-                        },
-                      ),
-                    ),
-                  ],
-                ),
-              ),
+              body: data.isLoading
+                  ? const HomeSkelet()
+                  : HomeContent(screenData: screenData, bloc: bloc),
               bottomNavigationBar: Container(
```

Comment:
let's move bottom navigation out of here. why? because in this case separate bottom navigation will be created for each screen. is it bad? I think yes. bottom navigation should be something screen-independent. let's move it to the app level, as we also have Scaffold here